### ISSUE TITLE
Crash on Fabric version 1.14+ caused by "Registry Object not present"

### ISSUE THREAD

**Issue by AngelTechnic** [KodoDenka/knavesneeds#12]
attempting to load up the current 1.20.1 version of Knave's Needs (1.14+) on Fabric results in the game crashing on startup.

https://pastebin.com/e5PKA7nq
i know nothing about how coding works but a quick perusal of this crash report, and this line in specific stands out to me:

> Caused by: java.lang.NullPointerException: Registry Object not present: knavesneeds:betterend/aeternium/chakram_head

**Comment by KodoDenka** [comment]
Yep, this is a goob on my side. I used BcLib in my support for BetterX's mods. And assumed that if BcLib was present it should load those items. Rolling out a hotfix ASAP.

**Comment by AngelTechnic** [comment]
it's doing it again, this time for a different reason

i am using the hotfix for this

https://pastebin.com/60MjFST0
